src/index.js

```diff
@@ -0,0 +1,34 @@
+import 'lodash';
+import './style.css';
+import {
+  addList, stausCheck, editList, removeTodo, removeCompleted,
+} from './status.js';
+
+const form = document.getElementById('addTodo');
+
+document.getElementById('showListItem').addEventListener('click', (e) => {
+  if (e.target.classList.contains('checkboX')) {
+    stausCheck(e);
+  }
+});
+
+document.getElementById('showListItem').addEventListener('click', (e) => {
+  if (e.target.classList.contains('fa-ellipsis-v')) {
+    editList(e);
+  }
+});
+
+document.getElementById('showListItem').addEventListener('click', (e) => {
+  if (e.target.classList.contains('fa-trash-o')) {
+    removeTodo(e);
```

Comment:
Inside index.js you call event listener for each function that is called on the same element. That element is repeating again and again. You can store it inside a variable and access it any number of times. For further info, you can use this link:[DRY, KISS, and YAGNI rules.](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/lesson_DRY_KISS_YAGNI.md)